src/@providers/docker/services/database.ts

```diff
@@ -239,15 +248,24 @@ export class DockerDatabaseService extends CrudResourceService<'database', Docke
           existingDeployment.labels?.['io.architect.arcctl.databaseVersion'],
       },
       exposed_ports: [{
-        port: 5432,
         target_port: 5432,
       }],
     });
 
+    const inspectionRes = await this.deploymentService.inspect(deployment.id);
```

Comment:
This code all looks heavily duplicated.

Reply:
Can you think of a better way to let docker dynamically select the host port to use so we don't need to use a fixed range and run port tests?